### ISSUE TITLE
PCI: Data co-pilot

### ISSUE THREAD

**Issue by danielabloch** [Parsely/wp-parsely#1647]
## Customer request
For joint Parse.ly/WordPressVIP customers, there is an interest in adding some sort of AI chatbot to the WP instance that would allow a user to ask a question like “What were the top articles in the Politics section by page views” and have the results show in the CMS.

## What is the customer need?
Ability to get answers quickly to specific questions. The common term used in the market right for now this sort of capability is "co-pilot" so that's a good placeholder for now.

**Comment by acicovic** [comment]
I agree that this looks like a good candidate for PCI. We need an API implementation for this though, which is outside of the scope of PCI. As such and if not already done, an issue/idea should be posted for this on the API side as well.

**Comment by acicovic** [comment]
@jbao, this is definitely something we could chew on, for ideas and feasibility.

**Comment by jbao** [comment]
Yeah, let's talk about that. Right now there have been some ideas about a chatbot in the Parsely dash, so on the API side, ideally we'd only have to develop one that can be used in both cases.
